src/mbgl/text/glyph_pbf.cpp

```diff
@@ -6,52 +6,55 @@
 #include <mbgl/text/glyph_store.hpp>
 #include <mbgl/text/glyph_set.hpp>
 #include <mbgl/util/exception.hpp>
-#include <mbgl/util/pbf.hpp>
 #include <mbgl/util/string.hpp>
 #include <mbgl/util/token.hpp>
 #include <mbgl/util/url.hpp>
 
+#include <protozero/pbf_reader.hpp>
+
 namespace {
 
 void parseGlyphPBF(mbgl::GlyphSet& glyphSet, const std::string& data) {
-    mbgl::pbf glyphs_pbf(reinterpret_cast<const uint8_t *>(data.data()), data.size());
-
-    while (glyphs_pbf.next()) {
-        if (glyphs_pbf.tag == 1) { // stacks
-            mbgl::pbf fontstack_pbf = glyphs_pbf.message();
-            while (fontstack_pbf.next()) {
-                if (fontstack_pbf.tag == 3) { // glyphs
-                    mbgl::pbf glyph_pbf = fontstack_pbf.message();
+    protozero::pbf_reader glyphs_pbf(data);
 
-                    mbgl::SDFGlyph glyph;
+    while (glyphs_pbf.next(1)) {
+        protozero::pbf_reader fontstack_pbf = glyphs_pbf.get_message();
+        while (fontstack_pbf.next(3)) {
+            protozero::pbf_reader glyph_pbf = fontstack_pbf.get_message();
 
-                    while (glyph_pbf.next()) {
-                        if (glyph_pbf.tag == 1) { // id
-                            glyph.id = glyph_pbf.varint();
-                        } else if (glyph_pbf.tag == 2) { // bitmap
-                            glyph.bitmap = glyph_pbf.string();
-                        } else if (glyph_pbf.tag == 3) { // width
-                            glyph.metrics.width = glyph_pbf.varint();
-                        } else if (glyph_pbf.tag == 4) { // height
-                            glyph.metrics.height = glyph_pbf.varint();
-                        } else if (glyph_pbf.tag == 5) { // left
-                            glyph.metrics.left = glyph_pbf.svarint();
-                        } else if (glyph_pbf.tag == 6) { // top
-                            glyph.metrics.top = glyph_pbf.svarint();
-                        } else if (glyph_pbf.tag == 7) { // advance
-                            glyph.metrics.advance = glyph_pbf.varint();
-                        } else {
-                            glyph_pbf.skip();
-                        }
-                    }
+            mbgl::SDFGlyph glyph;
 
-                    glyphSet.insert(glyph.id, glyph);
-                } else {
-                    fontstack_pbf.skip();
+            while (glyph_pbf.next()) {
+                switch (glyph_pbf.tag())
+                {
```

Comment:
Our code style mandates opening braces to be on the same line as the control statement.